language/vm/vm_runtime/src/loaded_data/function.rs

```diff
@@ -94,6 +94,19 @@ impl<'txn> FunctionReference<'txn> for FunctionRef<'txn> {
     }
 }
 
+impl<'txn> FunctionRef<'txn> {
+    pub fn pretty_string(&self) -> String {
+        let signature = self.signature();
+        format!(
+            "{}::{}({:?}){:?}",
+            self.module().name(),
+            self.name().as_str(),
+            signature.arg_types,
+            signature.return_types
+        )
+    }
+}
```

Comment:
`impl<'txn> fmt::Display for FunctionRef<'txn>`?

Reply:
I don't want to do that now if ok with you. I was starting that way but then I was not sure what usage of `Display` and `Debug` were around and did not want to overload those for now. 
The API defined here is eating quite a bit of info from a normal `Display` or `Debug`.
This is not the best way to go, I understand it, but I wanted to review at a different time if the usage of `Display` was good to move to what has been defined here